container_pipeline/workers/scan.py

```diff
@@ -53,12 +53,12 @@ def handle_job(self, job):
             self.logger.warning("Job data %s", str(self.job))
 
             self.set_buildphase_data(
-                build_phase_status='complete',
+                build_phase_status='failed',
                 build_phase_end_time=timezone.now()
             )
         else:
             self.set_buildphase_data(
-                build_phase_status='failed',
+                build_phase_status='complete',
                 build_phase_end_time=timezone.now()
```

Comment:
does this mean we were having it wrong? thanks for fixing this up.

Reply:
hah! yes, we were having it wrong earlier, fixed now. :+1: